File: 은지/LV1/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    for i in range(len(numbers)):
        for j in range(i+1, len(numbers)):
            x = numbers[i] + numbers[j]
            if x not in answer:
                answer.append(x)
    # 오름차순 정렬
    answer.sort()
    return answer

File: 준영/프로그래머스/LV1/두 개 뽑아서 더하기.js
function solution(numbers) {
  var answer = []

  for (let i = 0; i < numbers.length; i++) {
    for (let j = i; j < numbers.length; j++) {
      if (i != j) {
        var temp = numbers[i] + numbers[j]
        if (!answer.includes(temp)) {
          answer.push(temp)
        }
      }
    } 
  }

  answer.sort(function(a, b) {
    return a - b
  })
  return answer
}

solution([2,1,3,4,1])